### PR TITLE
Center data differently for scRNAseq

### PR DESCRIPTION
**Pull request by andrewram4287**

### CODE REVIEW

**Comment by aarmey** [comment]
Did this change the R2X results?

File: sccp/imports/scRNA.py [hunk 1]
```diff
@@ -216,7 +216,8 @@ def ThompsonXA_SCGenes(saveXA=False, offset=1.0):
             celltypeXA = xa.open_dataarray("/opt/andrew/scRNA_celltypeXA_NoOffset.nc")
         else:
             XA = xa.open_dataarray("/opt/andrew/scRNA_drugXA.nc")
-
+            
+    XA.values -= np.nanmean(XA.values, axis=(0,1), keepdims=True)
```
Comment (andrewram4287) [inline sccp/imports/scRNA.py:220]: @aarmey For some reason this transformation in the code creates many empty projections and cell types later in the code and greatly changes the results. 

I changed the normalization to  `XA.values /= np.nanmean(XA.values, axis=(0,1), keepdims=True)` and results dont create the same issue. 
Screen Shot 2023-03-16 at 2 13 34 PM
Screen Shot 2023-03-16 at 2 13 46 PM

Comment (aarmey) [reply sccp/imports/scRNA.py:220]: Aha! So this has an issue I was wondering about last night—this is being performed after we fill in the nans with 0. So, we need to do this earlier in the initial data processing.

**Comment by andrewram4287** [comment]
@aarmey Fixed the issue!

**Review by aarmey — APPROVED** [review]

**Pull request merged**